florist/api/db/entities.py

```diff
@@ -220,6 +223,44 @@ def set_client_metrics(
                 )
                 assert_updated_successfully(update_result)
 
+    async def set_server_log_file_path(self, log_file_path: str, database: AsyncIOMotorDatabase[Any]) -> None:
+        """
+        Save the server's log file path in the database under the current job's id.
+
+        :param log_file_path: (str) the file path to be saved in the database.
+        :param database: (motor.motor_asyncio.AsyncIOMotorDatabase) The database where the job collection is stored.
+        """
+        job_collection = database[JOB_COLLECTION_NAME]
+        self.server_log_file_path = log_file_path
+        update_result = await job_collection.update_one(
+            {"_id": self.id}, {"$set": {"server_log_file_path": log_file_path}}
+        )
+        assert_updated_successfully(update_result)
+
+    async def set_client_log_file_path(
+        self,
+        client_index: int,
+        log_file_path: str,
+        database: AsyncIOMotorDatabase[Any],
+    ) -> None:
+        """
+        Save the clients' log file path in the database under the given client index and current job's id.
+
+        :param client_index: (str) the index of the client in the job.
+        :param log_file_path: (str) the path oof the client's log file.
+        :param database: (motor.motor_asyncio.AsyncIOMotorDatabase) The database where the job collection is stored.
+        """
+        assert self.clients_info is not None, "Job has no clients."
```

Comment:
I maybe be missing something, but given the type of client_index, it doesn't seem like this should ever be None?